test/API/tselect.c

```diff
@@ -3189,7 +3189,7 @@ test_select_hyper_checker_board_dr__run_test(int test_num, const uint16_t *cube_
 
                     data_ok = test_select_hyper_checker_board_dr__verify_data(small_cube_buf_1, small_rank,
                                                                               edge_size, checker_edge_size,
-                                                                              expected_value, (hbool_t)TRUE);
+                                                                              expected_value, (bool)true);
```

Comment:
Cast can probably be removed here. Not sure that any compiler cares though

Reply:
Yeah, I thought about doing some additional cleanup work, but figured I'd leave that for a second pass later.